src/Compilers/Core/CodeAnalysisTest/FileSystem/PathUtilitiesTests.cs

```diff
@@ -259,7 +260,7 @@ public void TestContainsPathComponent()
                 PathUtilities.ContainsPathComponent(@"c:\Package\temp", "packages", ignoreCase: false));
         }
 
-        [Fact]
+        [ConditionalFact(typeof(WindowsOnly))]
```

Comment:
This is our standard way of disabling a test for Unix but it doesn't give us any information about why we had to disable the test. Unclear if this is a temporary problem or a permanent disable that's not any type of bug. 

Don't think we need to solve this in this PR. But would like to solve this in the soon-ish time frame as it looks like we'regoing to be moving a lot more code to run on Unix. Interested in ideas you have about this.